docs/schema.d.ts

```diff
@@ -58,13 +58,21 @@ export interface Pipeline {
    * package level (e.g. "a package's test and lint commands depend on build being
    * completed first").
    *
-   * Prefixing an item in dependsOn with a $ tells turbo that this pipeline task depends
-   * the value of that environment variable.
+   * DEPRECATED: Items prefixed with a `$` tells turbo that this pipeline task depends
+   * the value of that environment variable. This feature is deprecated and the `env` key
+   * should be used instead.
```

Comment:
This is the "just-in-time" documentation, let's avoid telling them about the deprecated feature? @mattpocock to second guess me, please.
```suggestion
```